content/code-style.md

```diff
@@ -0,0 +1,172 @@
+
+Collections should be named as a plural noun, in PascalCase. The name of the collection in the database (the first argument to the collection constructor) should be the same as the name of the JavaScript symbol.
+
+```js
+// Defining a collection
+Lists = new Mongo.Collection('Lists');
+```
+
+Fields in the database should be camelCased just like your JavaScript variable names.
+
+```js
+// Inserting a document with camelCased field names
+Widgets.insert({
+  myFieldName: 'Hello, world!',
+  otherFieldName: 'Goodbye.'
+});
+```
+
+<h3 id="methods-and-publications">Methods and publications</h3>
+
+Method and publication names should be camelCased and namespaced to a module or collection. For example, if you have a Method related to the Todos collection:
+
+```js
+// Naming a method
+Todos.methods.updateText = new ValidatedMethod({
+  name: 'Todos.methods.updateText',
+  // ...
+});
+```
+
+Note that this code sample uses the [ValidatedMethod package recommended in the Methods article](methods.html#XXX). If you aren't using that package, you can use the name as the property passed to `Meteor.methods`.
+
+Here's how this naming convention looks when applied to a publication:
+
+```js
+// Naming a publication
+Meteor.publish('Lists.public', function() {
+  // ...
+});
+```
+
+<h3 id="files-and-exports">Files, exports, and packages</h3>
+
+You should use the ES2015 `import` and `export` features to manage your code. This will let you better understand the dependencies between different parts of your code, and it will be easy to know where to look if you need to read the source code of a dependency.
+
+Each file in your app should represent one logical module. Avoid having catch-all utility modules that export a variety of unrelated functions and symbols. Often, this can mean that it's good to have one class, UI component, or collection per file, but there are cases where it is OK to make an exception, for example if you have a UI component with a small sub-component that isn't used outside of that file.
+
+When a file represents a single class or UI component, the file should be named the same as the thing it defines, with the same capitalization. So if you have a file that exports a class:
+
+```js
+export class ClickCounter { ... }
+```
+
+This class should be defined inside a file called `ClickCounter.js`. When you import it, it'll look like this:
+
+```js
+import ClickCounter from './ClickCounter.js';
+```
+
+Note that imports use relative paths, and include the file extension at the end of the file name.
+
+<h3 id="templates-and-components">Templates and components</h3>
+
+Often when you are dealing with templates or UI components, you'll have several closely coupled files to manage. They could be two or more of HTML, CSS, and JavaScript. In this case, we recommend putting the files together in the same directory with the same name:
+
+```
+# The listShow template from the Todos example app has 3 files:
+listShow.html
```

Comment:
OK new idea. How about instead of writing code, we can just recommend that people namespace with underscores: `Lists_show`.